2023-03-09T23:57:38.705+0800	INFO	instance/beat.go:640	Home path: [E:\code\云原生\git.imooc.com\dessple\pob] Config path: [E:\code\云原生\git.imooc.com\dessple\pob] Data path: [E:\code\云原生\git.imooc.com\dessple\pob\data] Logs path: [E:\code\云原生\git.imooc.com\dessple\pob\logs]
2023-03-09T23:57:38.706+0800	INFO	instance/beat.go:648	Beat ID: f781c3fd-0e4f-4897-a418-f8e76dba9b37
2023-03-09T23:57:38.707+0800	INFO	[beat]	instance/beat.go:976	Beat info	{"system_info": {"beat": {"path": {"config": "E:\\code\\云原生\\git.imooc.com\\dessple\\pob", "data": "E:\\code\\云原生\\git.imooc.com\\dessple\\pob\\data", "home": "E:\\code\\云原生\\git.imooc.com\\dessple\\pob", "logs": "E:\\code\\云原生\\git.imooc.com\\dessple\\pob\\logs"}, "type": "filebeat", "uuid": "f781c3fd-0e4f-4897-a418-f8e76dba9b37"}}}
2023-03-09T23:57:38.707+0800	INFO	[beat]	instance/beat.go:985	Build info	{"system_info": {"build": {"commit": "7aab6a9659749802201db8020c4f04b74cec2169", "libbeat": "7.9.3", "time": "2020-10-16T09:16:15.000Z", "version": "7.9.3"}}}
2023-03-09T23:57:38.707+0800	INFO	[beat]	instance/beat.go:988	Go runtime info	{"system_info": {"go": {"os":"windows","arch":"amd64","max_procs":12,"version":"go1.14.7"}}}
2023-03-09T23:57:38.788+0800	INFO	[beat]	instance/beat.go:992	Host info	{"system_info": {"host": {"architecture":"x86_64","boot_time":"2023-02-22T12:12:32.39+08:00","name":"DESKTOP-8K1IJ5A","ip":["fe80::2184:6ead:541:5295/64","169.254.23.90/16","fe80::9911:c532:6924:c868/64","172.31.144.1/20","fe80::5483:90c1:b30b:785e/64","169.254.244.105/16","fe80::35a9:1db8:a1f4:da6a/64","169.254.163.202/16","fe80::3318:2c57:4ae5:c3d8/64","192.168.142.1/24","fe80::bf70:9708:c69e:1b8/64","192.168.115.1/24","fd26:a241:87f4::d1f/128","fd26:a241:87f4:0:ce3d:e1bc:8bc0:100d/64","fd26:a241:87f4:0:6033:66cf:eaa9:9167/128","fd26:a241:87f4:0:d99c:5e73:3d70:9989/128","fd26:a241:87f4:0:f5d9:16bb:d810:9e93/128","fe80::ab2f:5bad:e890:55ea/64","192.168.1.128/24","fe80::99f4:b0e5:2e1d:d9a4/64","169.254.216.66/16","::1/128","127.0.0.1/8"],"kernel_version":"10.0.19041.2604 (WinBuild.160101.0800)","mac":["d8:bb:c1:4f:6c:c8","00:15:5d:fd:2b:60","e2:e1:a9:19:7a:8b","e0:e1:a9:19:7a:8b","00:50:56:c0:00:01","00:50:56:c0:00:08","e0:e1:a9:19:7a:8b","00:1a:7d:da:71:15"],"os":{"family":"windows","platform":"windows","name":"Windows 10 Pro","version":"10.0","major":10,"minor":0,"patch":0,"build":"19045.2604"},"timezone":"CST","timezone_offset_sec":28800,"id":"dc5c532d-a6b6-404d-a9b8-261c7f04c685"}}}
2023-03-09T23:57:38.788+0800	INFO	[beat]	instance/beat.go:1021	Process info	{"system_info": {"process": {"cwd": "E:\\code\\云原生\\git.imooc.com\\dessple\\pob", "exe": "D:\\filebeat-7.9.3-windows-x86_64\\filebeat.exe", "name": "filebeat.exe", "pid": 7196, "ppid": 1532, "start_time": "2023-03-09T23:57:38.529+0800"}}}
2023-03-09T23:57:38.788+0800	INFO	instance/beat.go:299	Setup Beat: filebeat; Version: 7.9.3
2023-03-09T23:57:38.788+0800	INFO	[publisher]	pipeline/module.go:113	Beat name: DESKTOP-8K1IJ5A
2023-03-09T23:57:38.789+0800	ERROR	fileset/modules.go:127	Not loading modules. Module directory not found: E:\code\云原生\git.imooc.com\dessple\pob\module
2023-03-09T23:57:38.789+0800	WARN	beater/filebeat.go:178	Filebeat is unable to load the Ingest Node pipelines for the configured modules because the Elasticsearch output is not configured/enabled. If you have already loaded the Ingest Node pipelines or are using Logstash pipelines, you can ignore this warning.
2023-03-09T23:57:38.789+0800	INFO	instance/beat.go:450	filebeat start running.
2023-03-09T23:57:38.789+0800	INFO	[monitoring]	log/log.go:118	Starting metrics logging every 30s
2023-03-09T23:57:38.789+0800	INFO	memlog/store.go:119	Loading data file of 'E:\code\云原生\git.imooc.com\dessple\pob\data\registry\filebeat' succeeded. Active transaction id=0
2023-03-09T23:57:38.789+0800	INFO	memlog/store.go:124	Finished loading transaction log file for 'E:\code\云原生\git.imooc.com\dessple\pob\data\registry\filebeat'. Active transaction id=13
2023-03-09T23:57:38.789+0800	WARN	beater/filebeat.go:381	Filebeat is unable to load the Ingest Node pipelines for the configured modules because the Elasticsearch output is not configured/enabled. If you have already loaded the Ingest Node pipelines or are using Logstash pipelines, you can ignore this warning.
2023-03-09T23:57:38.789+0800	INFO	[registrar]	registrar/registrar.go:109	States Loaded from registrar: 1
2023-03-09T23:57:38.789+0800	INFO	[crawler]	beater/crawler.go:71	Loading Inputs: 1
2023-03-09T23:57:38.789+0800	INFO	log/input.go:157	Configured paths: [E:\code\云原生\git.imooc.com\dessple\pob\*.log]
2023-03-09T23:57:38.789+0800	INFO	[crawler]	beater/crawler.go:141	Starting input (ID: 9579789163401830746)
2023-03-09T23:57:38.789+0800	INFO	[crawler]	beater/crawler.go:108	Loading and starting Inputs completed. Enabled inputs: 1
2023-03-09T23:58:08.792+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156,"time":{"ms":156}},"total":{"ticks":265,"time":{"ms":265},"value":265},"user":{"ticks":109,"time":{"ms":109}}},"handles":{"open":218},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":30140}},"memstats":{"gc_next":14088224,"memory_alloc":11305904,"memory_total":36376040,"rss":46411776},"runtime":{"goroutines":23}},"filebeat":{"events":{"added":1,"done":1},"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"output":{"type":"logstash"},"pipeline":{"clients":1,"events":{"active":0,"filtered":1,"total":1}}},"registrar":{"states":{"current":1,"update":1},"writes":{"success":1,"total":1}},"system":{"cpu":{"cores":12}}}}}
2023-03-09T23:58:38.791+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":217},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":60139}},"memstats":{"gc_next":14088224,"memory_alloc":11387160,"memory_total":36457296,"rss":40960},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":0}}},"registrar":{"states":{"current":1}}}}}
2023-03-09T23:59:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":219},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":90137}},"memstats":{"gc_next":14088224,"memory_alloc":11473568,"memory_total":36543704,"rss":24576},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":0}}},"registrar":{"states":{"current":1}}}}}
2023-03-09T23:59:38.791+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":221},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":120137}},"memstats":{"gc_next":15376400,"memory_alloc":11345400,"memory_total":36619832,"rss":290816},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":0}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:00:08.794+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":222},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":150142}},"memstats":{"gc_next":15376400,"memory_alloc":7758160,"memory_total":36693632,"rss":-151552},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":0}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:00:08.839+0800	INFO	log/harvester.go:299	Harvester started for file: E:\code\云原生\git.imooc.com\dessple\pob\micro.log
2023-03-10T00:00:09.839+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:143	Connecting to backoff(async(tcp://localhost:5044))
2023-03-10T00:00:09.839+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:00:09.839+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:00:15.823+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:00:15.823+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 1 reconnect attempt(s)
2023-03-10T00:00:15.823+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:00:15.823+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:00:22.961+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:00:22.961+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 2 reconnect attempt(s)
2023-03-10T00:00:22.961+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:00:22.961+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:00:32.807+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:00:32.807+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 3 reconnect attempt(s)
2023-03-10T00:00:32.807+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:00:32.807+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:00:38.804+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":241},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":180152}},"memstats":{"gc_next":15376400,"memory_alloc":8379488,"memory_total":37314960,"rss":913408},"runtime":{"goroutines":28}},"filebeat":{"events":{"active":6,"added":7,"done":1},"harvester":{"files":{"d0e9effe-62ac-4ef3-bbb4-26165b1a15f6":{"last_event_published_time":"2023-03-10T00:00:15.841Z","last_event_timestamp":"2023-03-10T00:00:15.841Z","name":"E:\\code\\云原生\\git.imooc.com\\dessple\\pob\\micro.log","read_offset":972,"size":486,"start_time":"2023-03-10T00:00:08.839Z"}},"open_files":1,"running":1,"started":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"filtered":1,"published":6,"retry":6,"total":7}}},"registrar":{"states":{"current":1,"update":1},"writes":{"success":1,"total":1}}}}}
2023-03-10T00:00:46.401+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:00:46.401+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 4 reconnect attempt(s)
2023-03-10T00:00:46.401+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:00:46.401+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:01:08.798+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":239},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":210146}},"memstats":{"gc_next":15376400,"memory_alloc":8474616,"memory_total":37410088,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"files":{"d0e9effe-62ac-4ef3-bbb4-26165b1a15f6":{"size":486}},"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:01:13.401+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:01:13.401+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 5 reconnect attempt(s)
2023-03-10T00:01:13.401+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:01:13.401+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:01:38.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":239},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":240137}},"memstats":{"gc_next":15376400,"memory_alloc":8559600,"memory_total":37495072,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:02:05.621+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:02:05.621+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 6 reconnect attempt(s)
2023-03-10T00:02:05.622+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:02:05.622+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:02:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":242},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":270137}},"memstats":{"gc_next":16244240,"memory_alloc":8173680,"memory_total":37588424,"rss":-3833856},"runtime":{"goroutines":29}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:02:38.794+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":243},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":300142}},"memstats":{"gc_next":16244240,"memory_alloc":8252248,"memory_total":37666992,"rss":69632},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:03:03.197+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:03:03.197+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 7 reconnect attempt(s)
2023-03-10T00:03:03.197+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:03:03.197+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:03:08.801+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":243},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":330148}},"memstats":{"gc_next":16244240,"memory_alloc":8337552,"memory_total":37752296,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:03:38.790+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":245},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":360138}},"memstats":{"gc_next":16244240,"memory_alloc":8407144,"memory_total":37821888},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:03:44.230+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:03:44.230+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 8 reconnect attempt(s)
2023-03-10T00:03:44.230+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:03:44.230+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:04:08.794+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":249},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":390142}},"memstats":{"gc_next":16240816,"memory_alloc":8158624,"memory_total":37916352,"rss":-786432},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:04:32.664+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:04:32.664+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 9 reconnect attempt(s)
2023-03-10T00:04:32.664+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:04:32.664+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:04:38.792+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":251},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":420140}},"memstats":{"gc_next":16240816,"memory_alloc":8252536,"memory_total":38010264,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:05:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":253},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":450137}},"memstats":{"gc_next":16240816,"memory_alloc":8328640,"memory_total":38086368},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:05:21.050+0800	INFO	log/harvester.go:330	File is inactive: E:\code\云原生\git.imooc.com\dessple\pob\micro.log. Closing because close_inactive of 5m0s reached.
2023-03-10T00:05:25.312+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:05:25.312+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 10 reconnect attempt(s)
2023-03-10T00:05:25.312+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:05:25.312+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:05:38.790+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":253},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":480138}},"memstats":{"gc_next":16240816,"memory_alloc":8414280,"memory_total":38172008},"runtime":{"goroutines":23}},"filebeat":{"events":{"active":1,"added":1},"harvester":{"closed":1,"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"filtered":1,"retry":3,"total":1}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:06:00.531+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:06:00.531+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 11 reconnect attempt(s)
2023-03-10T00:06:00.531+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:06:00.531+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:06:08.800+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":253},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":510148}},"memstats":{"gc_next":16160288,"memory_alloc":8127336,"memory_total":38266296,"rss":8192},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":6,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:06:19.045+0800	INFO	log/harvester.go:299	Harvester started for file: E:\code\云原生\git.imooc.com\dessple\pob\micro.log
2023-03-10T00:06:38.790+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":252},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":540138}},"memstats":{"gc_next":16160288,"memory_alloc":8467760,"memory_total":38606720,"rss":163840},"runtime":{"goroutines":28}},"filebeat":{"events":{"active":4,"added":4},"harvester":{"files":{"8ab6e02b-8244-4607-863f-1577b3e17448":{"last_event_published_time":"2023-03-10T00:06:19.045Z","last_event_timestamp":"2023-03-10T00:06:19.045Z","name":"E:\\code\\云原生\\git.imooc.com\\dessple\\pob\\micro.log","read_offset":1304,"size":1304,"start_time":"2023-03-10T00:06:19.045Z"}},"open_files":1,"running":1,"started":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"filtered":1,"published":3,"total":4}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:06:56.075+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:06:56.075+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 12 reconnect attempt(s)
2023-03-10T00:06:56.075+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:06:56.075+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:07:08.795+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":255},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":570143}},"memstats":{"gc_next":16160288,"memory_alloc":8548456,"memory_total":38687416,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:07:38.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":257},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":600137}},"memstats":{"gc_next":16160288,"memory_alloc":8624496,"memory_total":38763456,"rss":24576},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:07:59.704+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:07:59.704+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 13 reconnect attempt(s)
2023-03-10T00:07:59.704+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:07:59.704+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:08:08.791+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":258},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":630139}},"memstats":{"gc_next":16648736,"memory_alloc":8359176,"memory_total":38850176,"rss":16384},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:08:36.957+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:08:36.957+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 14 reconnect attempt(s)
2023-03-10T00:08:36.957+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:08:36.957+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:08:38.803+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":261},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":660151}},"memstats":{"gc_next":16648736,"memory_alloc":8445376,"memory_total":38936376,"rss":16384},"runtime":{"goroutines":29}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:09:08.800+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":256},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":690147}},"memstats":{"gc_next":16648736,"memory_alloc":8522632,"memory_total":39013632,"rss":-12288},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:09:27.516+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:09:27.516+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 15 reconnect attempt(s)
2023-03-10T00:09:27.516+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:09:27.516+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:09:38.797+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":258},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":720145}},"memstats":{"gc_next":16648736,"memory_alloc":8604864,"memory_total":39095864},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:10:08.803+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":258},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":750150}},"memstats":{"gc_next":16621616,"memory_alloc":8345600,"memory_total":39171760,"rss":24576},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:10:27.128+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:10:27.128+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 16 reconnect attempt(s)
2023-03-10T00:10:27.128+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:10:27.128+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:10:27.128+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:10:27.128+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:10:38.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":258},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":780137}},"memstats":{"gc_next":16621616,"memory_alloc":8434752,"memory_total":39260912,"rss":4096},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:11:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":256},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":810137}},"memstats":{"gc_next":16621616,"memory_alloc":8520680,"memory_total":39346840,"rss":16384},"runtime":{"goroutines":28}},"filebeat":{"harvester":{"open_files":1,"running":1}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:11:14.448+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:11:14.448+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 17 reconnect attempt(s)
2023-03-10T00:11:14.448+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:11:14.448+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:11:14.448+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:11:14.448+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:11:24.241+0800	INFO	log/harvester.go:330	File is inactive: E:\code\云原生\git.imooc.com\dessple\pob\micro.log. Closing because close_inactive of 5m0s reached.
2023-03-10T00:11:38.791+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":257},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":840139}},"memstats":{"gc_next":16621616,"memory_alloc":8600064,"memory_total":39426224,"rss":12288},"runtime":{"goroutines":23}},"filebeat":{"events":{"active":1,"added":1},"harvester":{"closed":1,"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"filtered":1,"retry":3,"total":1}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:12:02.821+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:12:02.821+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 18 reconnect attempt(s)
2023-03-10T00:12:02.821+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:12:02.821+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:12:02.821+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:12:02.821+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:12:08.791+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":260},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":870139}},"memstats":{"gc_next":16574432,"memory_alloc":8328808,"memory_total":39523776,"rss":12288},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:12:38.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":260},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":900137}},"memstats":{"gc_next":16574432,"memory_alloc":8399872,"memory_total":39594840},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:12:57.911+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp 127.0.0.1:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:12:57.911+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 19 reconnect attempt(s)
2023-03-10T00:12:57.911+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:12:57.911+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:12:57.911+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:12:57.911+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:13:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":261},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":930137}},"memstats":{"gc_next":16574432,"memory_alloc":8487304,"memory_total":39682272,"rss":4096},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:13:38.793+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":259},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":960140}},"memstats":{"gc_next":16574432,"memory_alloc":8555568,"memory_total":39750536},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:13:46.119+0800	ERROR	[publisher_pipeline_output]	pipeline/output.go:154	Failed to connect to backoff(async(tcp://localhost:5044)): dial tcp [::1]:5044: connectex: No connection could be made because the target machine actively refused it.
2023-03-10T00:13:46.119+0800	INFO	[publisher_pipeline_output]	pipeline/output.go:145	Attempting to reconnect to backoff(async(tcp://localhost:5044)) with 20 reconnect attempt(s)
2023-03-10T00:13:46.119+0800	INFO	[publisher]	pipeline/retry.go:219	retryer: send unwait signal to consumer
2023-03-10T00:13:46.119+0800	INFO	[publisher]	pipeline/retry.go:223	  done
2023-03-10T00:13:46.119+0800	INFO	[publisher]	pipeline/retry.go:213	retryer: send wait signal to consumer
2023-03-10T00:13:46.119+0800	INFO	[publisher]	pipeline/retry.go:217	  done
2023-03-10T00:14:08.789+0800	INFO	[monitoring]	log/log.go:145	Non-zero metrics in the last 30s	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156},"total":{"ticks":265,"value":265},"user":{"ticks":109}},"handles":{"open":263},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":990137}},"memstats":{"gc_next":16577264,"memory_alloc":8324528,"memory_total":39841424,"rss":28672},"runtime":{"goroutines":23}},"filebeat":{"harvester":{"open_files":0,"running":0}},"libbeat":{"config":{"module":{"running":0}},"pipeline":{"clients":1,"events":{"active":9,"retry":3}}},"registrar":{"states":{"current":1}}}}}
2023-03-10T00:14:15.854+0800	INFO	beater/filebeat.go:515	Stopping filebeat
2023-03-10T00:14:15.854+0800	INFO	beater/crawler.go:148	Stopping Crawler
2023-03-10T00:14:15.854+0800	INFO	beater/crawler.go:158	Stopping 1 inputs
2023-03-10T00:14:15.854+0800	INFO	[crawler]	beater/crawler.go:163	Stopping input: 9579789163401830746
2023-03-10T00:14:15.854+0800	INFO	input/input.go:136	input ticker stopped
2023-03-10T00:14:15.854+0800	INFO	beater/crawler.go:178	Crawler stopped
2023-03-10T00:14:15.854+0800	INFO	[registrar]	registrar/registrar.go:132	Stopping Registrar
2023-03-10T00:14:15.854+0800	INFO	[registrar]	registrar/registrar.go:166	Ending Registrar
2023-03-10T00:14:15.854+0800	INFO	[registrar]	registrar/registrar.go:137	Registrar stopped
2023-03-10T00:14:15.855+0800	INFO	[monitoring]	log/log.go:153	Total non-zero metrics	{"monitoring": {"metrics": {"beat":{"cpu":{"system":{"ticks":156,"time":{"ms":156}},"total":{"ticks":265,"time":{"ms":265},"value":265},"user":{"ticks":109,"time":{"ms":109}}},"handles":{"open":262},"info":{"ephemeral_id":"ff77ef35-235f-4ca3-b380-81450321e53f","uptime":{"ms":997203}},"memstats":{"gc_next":16577264,"memory_alloc":8400552,"memory_total":39917448,"rss":43372544},"runtime":{"goroutines":13}},"filebeat":{"events":{"active":12,"added":14,"done":2},"harvester":{"closed":2,"open_files":0,"running":0,"started":2}},"libbeat":{"config":{"module":{"running":0}},"output":{"type":"logstash"},"pipeline":{"clients":0,"events":{"active":9,"filtered":5,"published":9,"retry":54,"total":14}}},"registrar":{"states":{"current":1,"update":2},"writes":{"success":2,"total":2}},"system":{"cpu":{"cores":12}}}}}
2023-03-10T00:14:15.855+0800	INFO	[monitoring]	log/log.go:154	Uptime: 16m37.2039106s
2023-03-10T00:14:15.855+0800	INFO	[monitoring]	log/log.go:131	Stopping metrics logging.
2023-03-10T00:14:15.855+0800	INFO	instance/beat.go:456	filebeat stopped.
